controllers/controller.js

```diff
@@ -28,9 +29,20 @@ const findSpecificArticle = (request, response, next) => {
     .catch(next);
 };
 
+const findArticleComments = (request, response, next) => {
+  if (isNaN(Number(request.params.article_id))) {
+    response.status(400).send({ msg: "400 - Bad request" });
+  }
```

Comment:
this shows a lot of gusto to have come up with a way to handle this error - but - almost always a `400` error is something you don't need to handle for yourself, psql will throw the error for you 

Take this `if` statement out and allow the invalid `article_id` to go into the query - you will find that psql will throw an error back to your catch block, if you console.log the error it should have a code of `22P02` and you can use error handlers to just send back the right response of `400` when that occurs 

Let psql do the hard work for you